### PR TITLE
SonarCloud Issue

### PR DESCRIPTION
**Pull request by rcmsoftwaredevelopment**
https://hackney.atlassian.net/jira/software/projects/MTTL/boards/47?assignee=624c20c6a629c30068a81020&selectedIssue=MTTL-2956

Resolve SonarCloud Issues

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```
Comment (humulla) [inline .idea/.gitignore:1]: What is this used for?

File: .idea/housing-search-listener.iml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (humulla) [inline .idea/housing-search-listener.iml:1]: What is this used for?

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (humulla) [inline .idea/vcs.xml:1]: What is this used for?

File: .idea/.gitignore (hunk 1 above)
Comment (rcmsoftwaredevelopment) [reply .idea/.gitignore]: File created from IDE.  File has been removed.

File: .idea/vcs.xml (hunk 3 above)
Comment (rcmsoftwaredevelopment) [reply .idea/vcs.xml]: File created from IDE.  File has been removed.

File: .idea/housing-search-listener.iml (hunk 2 above)
Comment (rcmsoftwaredevelopment) [reply .idea/housing-search-listener.iml]: File created from IDE.  File has been removed.

**Review by humulla — APPROVED** [review]

**Pull request merged**